pynamodb/settings.py

```diff
@@ -1,8 +1,8 @@
-import imp
 import logging
 import os
 import warnings
 from os import getenv
+from importlib.machinery import SourceFileLoader
```

Comment:
We still maintain Python 2.7 compatibility in PynamoDB 4.x. Maybe add something to pynamodb.compat?

Reply:
Thanks! I've updated the _compat module.